CHANGELOG.rst

```diff
@@ -1,5 +1,41 @@
 Changelog
 =========
+1.3 (2019-09-30)
+------------------
+MLflow 1.3.0 includes several major features and improvements:
+
+Features:
+
+- The Python client now supports logging & loading models using TensorFlow 2.0 (#1872, @juntai-zheng)
```

Comment:
w00t!

Reply:
[just in time too](https://github.com/tensorflow/tensorflow/releases/tag/v2.0.0) :-)  cc @juntai-zheng